arifmetic.py

```diff
@@ -1,23 +1,23 @@
-# Написать функцию arithmetic, принимающую 3 аргумента: первые 2 - числа, третий - операция
-# , которая должна быть произведена над ними. Если третий аргумент +, сложить их; если —,
-# то вычесть; * — умножить; / — разделить (первое на второе). В остальных случаях вернуть строку
-# "Неизвестная операция".
 
-def arithmetic(a,b,c):  # todo add whitespaces
+
+def arithmetic(a, b, c):
```

Comment:
Use something self-describing like:
```suggestion
def arithmetic(a, b, operation):
```

Reply:
Let's start using type hints:
```suggestion
def arithmetic(a: float, b: float, operation: str) -> float:
```